contracts/consensus/Consensus.sol

```diff
@@ -116,6 +117,12 @@ contract Consensus is MasterCopyNonUpgradable, CoreLifetimeEnum, MosaicVersion,
     /** Coinbase split per mille */
     uint256 public coinbaseSplitPerMille;
 
+    /** Gas price to calculate reward */
+    uint256 public feeGasPrice;
```

Comment:
How this value can be set? 
I can't find initialization in the setup? should it be constant for now?

Reply:
Same comment for feeGasLimit

Reply:
These two parameters will be provided and managed by the techgov. The logic regarding updating these values will later be finalized in either setup function or some other way.
In current implementation initialization is done in setup function.

Detailed discussion on the ticket: https://github.com/mosaicdao/mosaic-1/issues/115